centraleventprocessor/configs/default.json

```diff
@@ -0,0 +1,111 @@
+{{- $kafkaHost := ( .Values.config.kafka_host | replace "$release_name" .Release.Name ) -}}
+{{- $mongodbHost := ( .Values.config.mongo_host | replace "$release_name" .Release.Name ) -}}
+{{- $centralLedgerAdminHost := printf "%s-%s" .Release.Name .Values.config.central_ledger_admin_host -}}
+{{- $centralLedgerApiHost := printf "%s-%s" .Release.Name .Values.config.central_ledger_api_host -}}
```

Comment:
The above two lines do not match the master branch code base.

Have a look at the code from master branch as it follows the same pattern as central-ledger components:
```
{{- $centralLedgerAdminHost := ( .Values.config.central_ledger_admin_host | replace "$release_name" .Release.Name ) -}}
{{- $centralLedgerApiHost := ( .Values.config.central_ledger_admin_host | replace "$release_name" .Release.Name ) -}}
```

Refer to the PR that @vgenev closed -> https://github.com/mojaloop/helm/pull/124/files#diff-9b9b50f835536aba43c93c817a1bfd9cL4